src/utils/image-manager.ts

```diff
@@ -102,12 +102,7 @@ export const isImageFresh = async (deviceType: string, version: string) => {
 		return false;
 	}
 
-	const balena = getBalenaSdk();
-	const lastModifiedDate = await balena.models.os.getLastModified(
-		deviceType,
-		version,
-	);
-	return lastModifiedDate < createdDate;
+	return true;
```

Comment:
I guess you could even delete this whole function since there is no point on fetching the image's creation date in the cache folder.